### DISCUSSION TITLE
Don't revalidate if request response with No content

### DISCUSSION THREAD

**Discussion by betoquiroga** [vercel/next.js#29781] [Ideas]
### Describe the feature you'd like to request

Sometimes our API is down. If revalidate runs in our Next app, the content "disappear" since there's no info to show from API.

### Describe the solution you'd like

I think the title is the possible option: Don't revalidate the content in some cases, maybe we can set the response code to run revalidate, form example 200 or 201.  Any other response code won't execute the revalidate.

### Describe alternatives you've considered

We can have a config object to set these options to revalidate or don't

**Comment by ijjk** [comment]
Hi, you can currently prevent the revalidation cache from being updated when an upstream response isn't correct by throwing an error inside of `getStaticProps`. When this function errors the cache entry is not updated and will retry in the next revalidation.
